src/6.3.01/micro_bin/microphysics.f90

```diff
@@ -1194,29 +1203,29 @@ Subroutine adj1_bin (m1,m2,m3,rtp,micro)
             endif
          enddo
          do kr = 1,nkr
-            if (micro%ffcd(k,i,j,kr)*col < rxmin) micro%ffcd(k,i,j,kr) = 0.
+            if (micro%ffcd(k,i,j,kr)*col < 1.e-12) micro%ffcd(k,i,j,kr) = 0.
```

Comment:
Why have we gone from `rxmin` to 1.e-12 here? Can we start to move toward this being a variable and/or macro?

Reply:
Is this a question for Steve? I agree with your comment

Reply:
Just talked to Steve - he said this is only for the bin scheme and the rxmix (set to 1.e-9) results in the bin model crashing so he changed these manually to be 1.e-12. It could be made into a separate rxmix variable specific to the bin scheme in a future release.